dist/gmusic-theme.js

```diff
@@ -1008,155 +1013,155 @@ Converter.prototype.getValues = function(space) {
 
 module.exports = convert;
 },{"./conversions":3}],5:[function(require,module,exports){
-module.exports = {
```

Comment:
If I run the build sequence in stages, it seems to be browserify causing the difference

https://github.com/Matt-Esch/virtual-dom/issues/356